build.sbt

```diff
@@ -29,6 +29,12 @@ lazy val joda = (project in file("modules/joda")).
   dependsOn(core).
   dependsOn(core % "test->test") // In order to reuse the date/time related scalacheck generators.
 
+// play 2.5 is only published for Scala 2.11
+lazy val play = (project in file("modules/play")).
+  settings(settings).
+  settings(crossScalaVersions ~= { oldVersions => oldVersions.filter(_.startsWith("2.11")) }).
```

Comment:
The `scalaVersion` key is still set to `2.12.2` here. You should change this line to something like:

```scala
  settings(
    scalaVersion := "2.11.11",
    crossScalaVersions ~= { oldVersions => oldVersions.filter(_.startsWith("2.11")) })
```